server/services/wellcomecollection-api.js

```diff
@@ -13,7 +13,7 @@ export async function getWork(id: string): Promise<Work> {
 
 export async function getWorks(query: string, page: string): Promise<Work> {
   return await superagent.get(`${baseUri}/works`)
-    .query({query, includes: 'identifiers', page, pageSize: 96})
+    .query({query, includes: 'identifiers,thumbnail', page, pageSize: 96})
```

Comment:
👾

Reply:
This string gets appended to the URL wholesale so needs to be spaceless, e.g. `https://api.wellcomecollection.org/catalogue/v0/works/a23exgqh?includes=thumbnail,identifiers`